.github/workflows/r-ci.yml

```diff
@@ -108,3 +106,5 @@ jobs:
       - name: Coverage
         if: ${{ matrix.os == 'ubuntu-latest' && matrix.covr == 'yes' }}
         run: cd apis/r && tools/r-ci.sh coverage
+        env:
+          R_COVR: 'true'
```

Comment:
The `covr` package guarantees setting `COVR` which lets you test for it in a test file. That appears to be enough in my use cases.